tests/abstract4-test.toit

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2021 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/LICENSE file.
+
+import expect show *
+
+class A:
+  foo: return 1
+
+abstract class B extends A:
+  abstract foo x=null
+
+class C extends B:
+  foo x=null:
+    return super
+
+class D:
+  foo:
+    return 33
+
+confuse x: return x
```

Comment:
I think we have a better way of confusing in `tests/toitp/confuse.toit`. Maybe we should use that in more places?

Reply:
I think we should eventually go through all `confuse` methods and make it use something similar.
I'm not too keen on using something from the `toitp` directory.
In another PR...